src/store/tests/vehicle-details/cat-b/vehicle-details.cat-b.reducer.ts

```diff
@@ -37,6 +37,14 @@ export const vehicleDetailsReducer = createReducer(
     ...state,
     gearboxCategory: null,
   })),
+  on(vehicleDetailsActions.MotEvidenceChanged, (state, { motEvidence }): VehicleDetails => ({
+    ...state,
+    motEvidence,
+  })),
+  on(vehicleDetailsActions.MotEvidenceProvidedToggled, (state, { motEvidenceProvided }): VehicleDetails => ({
+    ...state,
+    motEvidenceProvided,
+  })),
```

Comment:
Where are the make, model, testExpiry and previously searched for fields? Do they not exist in the schema? If not we need to add them and implement necessary actions/reducers/logic